docs/src/components/footer/EmailSubscribe.tsx

```diff
@@ -0,0 +1,135 @@
+import InputBase from '@material-ui/core/InputBase';
+import Button from '@material-ui/core/Button';
+import CheckCircleRoundedIcon from '@material-ui/icons/CheckCircleRounded';
+
+const Form = styled('form')(({ theme }) => ({
+  display: 'inline-flex',
+  borderRadius: theme.shape.borderRadius,
+  overflow: 'hidden',
+}));
+
+function searchParams(params: any) {
+  return Object.keys(params)
+    .filter((key) => params[key] != null)
+    .map((key) => `${encodeURIComponent(key)}=${encodeURIComponent(params[key])}`)
+    .join('&');
+}
+
+export default function EmailSubscribe({ sx }: { sx?: SxProps<Theme> }) {
+  const [form, setForm] = React.useState<'sent' | 'loading' | 'pristine'>('pristine');
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setForm('loading');
+    const target = event.target as {
+      elements?: { email?: { value: string } };
+    };
+    await fetch(
+      'https://f0433e60.sibforms.com/serve/MUIEAMMuohK-i-XUkJaUj3Lq3zr3rVeAPmgssEBsyiTktpqrImORJiFMQ1PLfZ1W1PGb-FzKhlfuPWlLNfx90j5R2qC7C219ec8AVcBsxlIRDG5znwaXr6gzAyth6W93bLiK4otXL_iBLFV43QqHrKZKORXA0LGq6seXbasTiAHh5EtqWFGK2zw8mlwYssGnIT_7ZZXiWC_iqubZ',
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/x-www-form-urlencoded',
+        },
+        mode: 'no-cors',
+        body: searchParams({
+          EMAIL: target.elements?.email?.value,
+          email_address_check: '',
+          locale: 'en',
+        }),
+      },
+    );
+    setForm('sent');
+  };
+  if (form === 'sent') {
+    return (
+      <Alert
+        severity="success"
+        sx={{
+          maxWidth: { sm: 400 },
+          bgcolor: (theme) =>
+            theme.palette.mode === 'dark'
+              ? theme.palette.primaryDark[700]
+              : theme.palette.success[50],
+          ...sx,
+        }}
+        iconMapping={{
+          success: (
+            <CheckCircleRoundedIcon
+              fontSize="small"
+              sx={{
+                color: (theme: Theme) =>
+                  theme.palette.mode === 'dark'
+                    ? theme.palette.success[500]
+                    : theme.palette.success[700],
+              }}
+            />
+          ),
+        }}
+      >
+        <AlertTitle sx={{ typography: 'body2', fontWeight: 600 }}>
+          Thanks! Check your email.
+        </AlertTitle>
+        You should get a <strong>confirmation email</strong> soon. Open it up and confirm your email
+        address so that we can keep you up to date.
+      </Alert>
+    );
+  }
+  return (
+    <Form
+      onSubmit={handleSubmit}
+      sx={{
+        width: { xs: '100%', sm: 'auto' },
+        maxWidth: 360,
+        ...sx,
+      }}
+    >
+      <InputBase
+        placeholder="Enter your email"
+        name="email"
+        type="email"
+        inputProps={{
+          required: true,
+          'aria-label': 'Enter your email',
```

Comment:
Visible label first. Once entered, the user will no longer be able to determine what the input is for.